p2p/pex/reactor.go

```diff
@@ -16,22 +17,72 @@ var (
 	_ p2p.Wrapper     = (*protop2p.PexMessage)(nil)
 )
 
+// TODO: Consolidate with params file.
 const (
-	maxAddresses   uint16 = 100
-	resolveTimeout        = 3 * time.Second
+	// the minimum time one peer can send another request to the same peer
+	minReceiveRequestInterval = 300 * time.Millisecond
+
+	// the maximum amount of addresses that can be included in a response
+	maxAddresses uint16 = 100
+
+	// allocated time to resolve a node address into a set of endpoints
+	resolveTimeout = 3 * time.Second
+
+	// How long to wait when there are no peers available before trying again
+	noAvailablePeersWaitPeriod = 1 * time.Second
+
+	// indicates the ping rate of the pex reactor when the peer store is full.
+	// The reactor should still look to add new peers in order to flush out low
+	// scoring peers that are still in the peer store
+	fullCapacityInterval = 10 * time.Minute
 )
 
 // ReactorV2 is a PEX reactor for the new P2P stack. The legacy reactor
 // is Reactor.
 //
 // FIXME: Rename this when Reactor is removed, and consider moving to p2p/.
+//
+// The peer explorer or PEX reactor supports the peer manager by sending
```

Comment:
```suggestion
// The peer exchange or PEX reactor supports the peer manager by sending
```

Reply:
Lol that's right. I keep forgetting that it's the peer exchange not the peer explorer 😄